src/cpu/rv64/rvv_postops.hpp

```diff
@@ -79,6 +80,27 @@ struct rvv_postops_t {
                 CHECK(bin_pd->create_primitive(bin_prim, engine));
                 post_op_primitives_.push_back(bin_prim);
 
+            } else if (po.is_eltwise()) {
+                eltwise_desc_t po_desc;
+                po_desc.primitive_kind = primitive_kind::eltwise;
+                po_desc.prop_kind = prop_kind::forward_inference;
+                po_desc.alg_kind = po.eltwise.alg;
+                po_desc.alpha = po.eltwise.alpha;
+                po_desc.beta = po.eltwise.beta;
+                po_desc.src_desc = dst_md;
+                po_desc.dst_desc = dst_md;
+
+                auto empty_attr = dnnl_primitive_attr();
+                primitive_desc_iterator_t it(engine,
+                        reinterpret_cast<const op_desc_t *>(&po_desc),
+                        &empty_attr, nullptr);
+                if (++it == it.end()) return status::unimplemented;
+
+                std::shared_ptr<primitive_desc_t> elt_pd = *it;
+                std::shared_ptr<primitive_t> elt_prim;
+                CHECK(elt_pd->create_primitive(elt_prim, engine));
+                post_op_primitives_.push_back(elt_prim);
```

Comment:
This would be bad for performance if there are more than 1 post-op in the chain. I suggest to reserve `len()` elements in the vector before getting into the for loop.

Besides, I'm not sure this is the right approach. Optimized version want to do such fusions on registers, not on memory which is not different from the case when the user does it not applying post-ops but applying plain primitives. For non-optimized approach there's already `ref_post_ops_t` struct that mostly handles everything for you. Feel free to inspect existing reference implementations with this struct and let me know you opinion on this one.